precompile/allow_list.go

```diff
@@ -23,9 +23,12 @@ const (
 )
 
 var (
-	AllowListNoRole  AllowListRole = AllowListRole(common.BigToHash(big.NewInt(0))) // No role assigned - this is equivalent to common.Hash{} and deletes the key from the DB when set
-	AllowListEnabled AllowListRole = AllowListRole(common.BigToHash(big.NewInt(1))) // Deployers are allowed to create new contracts
-	AllowListAdmin   AllowListRole = AllowListRole(common.BigToHash(big.NewInt(2))) // Admin - allowed to modify both the admin and deployer list as well as deploy contracts
+	// No role assigned - this is equivalent to common.Hash{} and deletes the key from the DB when set
+	AllowListNoRole AllowListRole = AllowListRole(common.BigToHash(big.NewInt(0)))
```

Comment:
nit: could we use `comon.Big0`, `Big1` and `Big2`?
(are these equivalent to `common.Hash{0}, common.Hash{1}, and common.Hash{2}`)?